src/types/utils/viewForwardedRef.ts

```diff
@@ -1,6 +1,6 @@
 import type {ForwardedRef} from 'react';
 import type {View} from 'react-native';
 
-const viewForwardedRef = (ref: ForwardedRef<View | HTMLDivElement>) => ref as ForwardedRef<View>;
+const viewForwardedRef = (ref: ForwardedRef<View | HTMLElement>) => ref as ForwardedRef<View>;
```

Comment:
Why these changes in utils?

Reply:
`HTMLELement` is more generic then `HTMLDivElement` and this function is used to cast to `View` so I changed it to accept other HTML elements as well. It was more useful for the `textRef` where there can be more than one html element (`HTMLDivElement` or `HTMLFormElement` for example).